UI/TiltAnalogSettingsScreen.cpp

```diff
@@ -35,6 +35,8 @@ void TiltAnalogSettingsScreen::CreateViews() {
 	settings->Add(new ItemHeader(co->T("Invert Axes")));
 	settings->Add(new CheckBox(&g_Config.bInvertTiltX, co->T("Invert Tilt along X axis")));
 	settings->Add(new CheckBox(&g_Config.bInvertTiltY, co->T("Invert Tilt along Y axis")));
+	static const char* tiltMode[] = { "Screen parallel to ground", "Screen orthogonal to ground", "Auto-switch" };
```

Comment:
I know a bunch of people who have trouble with terms like orthogonal (or perpendicular) and parallel.  Not sure what's better, maybe "Screen level with ground" and "Screen upright from ground"?  Still seems confusing.

-[Unknown]

Reply:
I find level and upright more confusing personally, but I guess it's a matter of native language.
What about "horizontal" and "vertical"? Might be a confused with landscape/portrait tho'.

Reply:
Yeah, colloquial English just doesn't cover this well.  I guess people will figure it out.

-[Unknown]

Reply:
I think "perpendicular" would be easier to understand than "orthogonal".

Reply:
"Perpendicular" it's more common indeed, I wonder if "Screen aligned to ground" and "Screen at right angle to ground" would be simpler.

I can't quite tell what is easy and what is hard with words: some english word with a latin root are unusual to someone from USA or UK, but at the same time look almost obvious in Italy, Spain or France as they sound alike the everyday one in those languages.

Reply:
I like those last two suggestions myself.  They're a bit long but "right angle to ground" would make sense to the vast majority of English speakers I know.  The other might not make sense to as many immediately, but presented with both they'd be obvious.

-[Unknown]